dfencoder/autoencoder.py

```diff
@@ -164,8 +164,9 @@ def __init__(self,
                  run=None,
                  progress_bar=True,
                  n_megabatches=1,
-                 scaler='standard',
```

Comment:
Removing the `scalar` constructor arg is a breaking change for Morpheus because it's used by `DFPTraining` stage:
https://github.com/nv-morpheus/Morpheus/blob/branch-23.01/examples/digital_fingerprinting/production/morpheus/dfp/stages/dfp_training.py#L47

Need Morpheus PR to update accordingly once this is merged.

Reply:
Changed it back to maintain backward compatibility. Added a comment to help distinguish the two scalers.